mac/hid.c

```diff
@@ -763,64 +859,38 @@ hid_device * HID_API_EXPORT hid_open_path(const char *path)
 
 static int set_report(hid_device *dev, IOHIDReportType type, const unsigned char *data, size_t length)
 {
-	const unsigned char *data_to_send = data;
```

Comment:
@todbot:
I mean this function and all of the below shouldn't be changed at all
see https://github.com/libusb/hidapi/commit/083223e77952e1ef57e6b77796536a3359c1b2a3 and https://github.com/libusb/hidapi/commit/d64f6c6a16ac0468e9066767080c818c995c37b1

Reply:
yeah, I guess it was

a hint:
- checkout  fengj2006's branch
- save a diff only: `git diff HEAD~1 > diff.diff`
so it only contain a relevant changes
- checkout to `origin/master`
- apply diff: `git apply diff.diff`

Reply:
oh, you've already done it :)